Riot/Modules/TabBar/MasterTabBarController.m

```diff
@@ -75,8 +75,8 @@ @implementation MasterTabBarController
 
 - (HomeViewController *)homeViewController
 {
-    HomeViewControllerWithBannerWrapperViewController *wrapperVC = [self viewControllerForClass:HomeViewControllerWithBannerWrapperViewController.class];
-    return wrapperVC.homeViewController;
+    UIViewController *wrapperVC = [self viewControllerForClass:HomeViewControllerWithBannerWrapperViewController.class];
+    return [(HomeViewControllerWithBannerWrapperViewController *)wrapperVC homeViewController];
```

Comment:
How does it matter?

Reply:
Oh, it doesn't, it's just to fix a warning. Otherwise I would need to cast the result and the line would be too long 🙃

Reply:
Ah ok cool.